R/wrappers.R

```diff
@@ -29,40 +29,40 @@
 #'   category = "something"
```

Comment:
same for this file. only whitespace changes?

Reply:
Some changes at  386.  Not entirely sure what all the white space changes are.  I'm using Rstudio `1.4.1103`, could there be some line return formatting causing this?

Reply:
Rstudio has some formating options, removing trailing whitespace options, ensure empty line at the end of the file. For personal projects I usually stick with them, but all this unrelated noise make this pull request harder to read.
I'm not against the changes and from a quick glance they seem to improve the code, I just don't want to review all the lines of unrelated changes looking for possible issues in the new and changed code 😄

Reply:
> Rstudio has some formating options, removing trailing whitespace options, ensure empty line at the end of the file. For personal projects I usually stick with them, but all this unrelated noise make this pull request harder to read.

Agreed.  I feel like this isn't going to be an isolated thing.  Temping now to make one big PR of just white-noise?

Reply:
First of, that's not my call to make (poking @ycphs) and ideally it would be a decision unrelated to an IDE configuration.
Personally, I would prefer to introduce such changes only when applying code changes. Iirc correctly, such whitespace related bulk commits make git bisecting and blaming harder, but _I didn't check that_.
Finally, I think that this should not be a blocker for this pull request, but I won't volunteer right now to review it. (And I know projects, where such pull request would be closed without any hesitation.)

Reply:
I totally agree with @JanMarvin. It is hard to read, but I will merge the PR. The next time it would be great if only the rows with actual changes is pushed.